src/data/overfor/overforSagas.ts

```diff
@@ -0,0 +1,37 @@
+import { call, put, takeEvery } from "redux-saga/effects";
+import {
+  moterOverfort,
+  OVERFOR_MOTER_FORESPURT,
+  overforerMoter,
+  OverforMoterAction,
+  overforMoterFeilet,
+  resetOverforing,
+} from "./overfor_actions";
+import { post } from "../../api";
+import { ISDIALOGMOTE_ROOT, SYFOMOTEADMIN_ROOT } from "../../utils/apiUrlUtil";
+
+export function* overforMoter(action: OverforMoterAction) {
+  yield put(overforerMoter());
+  try {
+    const { moteUuids, dialogmoteUuids } = action;
+    if (moteUuids.length > 0) {
+      yield call(post, `${SYFOMOTEADMIN_ROOT}/v2/actions/moter/overfor`, {
+        moteUuidListe: moteUuids,
+      });
+    }
+    if (dialogmoteUuids.length > 0) {
+      yield call(post, `${ISDIALOGMOTE_ROOT}/v2/dialogmote/overta`, {
+        dialogmoteUuids: dialogmoteUuids,
+      });
+    }
```

Comment:
Betyr dette at det nå kan bli fyrt av 2 post-kall i denne sagaen nå?
I så fall: Hvordan blir resultatet for brukeren om det ene kallet blir 200-ok og det andre f.eks 500?
Koden kaster da overforMoterFeilet og setter sendingFeilet til true ser, men kan ikke se at det brukes til noe.

Reply:
sendingFeilet blir brukt til å vise feilmelding i EnhetensMoter (overtaMoterFeilet). Men du har rett i at det nå er potensielt to POST-kall (i stedet for ett som det var tidligere). Så bør nok tenke på hvordan vi skal håndtere at ett av de kan feile 🤔

Reply:
Det greieste er kanskje å splitte i to actions, én for å overta dialogmøteinnkallinger og én for møteplanlegger-møter. Med hvert sitt sendingFeilet-flagg osv. Og så vise hver sin eventuelle feilmelding/suksessmelding?